### PR TITLE
Tcs 128 subtablefiltered logmsg fix

### PR DESCRIPTION
**Pull request by ThomasGillespie**
Changed subscription log messages to change based met conditions

### CODE REVIEW

File: code/common/pubsub.q [hunk 1]
```diff
@@ -154,7 +154,16 @@ init:{[t]
 
 // Allow a non-kdb+ subscriber to subscribe with strings for complex conditions - return table name and schema to subscriber
 .ps.subtablefiltered:{[tab;filters;columns]
-  .lg.o[`subtablefiltered;"Received a subscription to ",$[count tab;tab;"all tables"]," for filters: ",filters," and columns: ",columns];
+  //creates empty strings to be edited under certain conditions
+  bfilters:"";
+  bcolumns:"";
+  
+  //conditions met to edit strings accordingly
+  if[count filters;bfilters:" filters: "];
```
Comment (dmitchard) [inline code/common/pubsub.q:162]: Would using `any` here for the condition work?

**Review by dmitchard — APPROVED** [review]

File: code/segmentedtickerplant/datastripe.q [hunk 2]
```diff
@@ -51,7 +51,9 @@ subsegment:{[tbl;segid];
      if[tbl in ignoredtables;
       .lg.o[`sub;m:"Table ",string[tbl]," is to be ignored for segment ",string[segid],""];
       :()];
-     .ps.subtablefiltered[string[tbl];filter;""]
+      
+     //if a filter has been provided use subtablefiltered, if not use subtable for all syms
+     $[count filter;.ps.subtablefiltered[string[tbl];filter;""];.ps.subtable[string[tbl];"
```
Comment (jonathonmcmurray) [inline code/segmentedtickerplant/datastripe.q:56]: are there some missing `]` at the end of this line?

Comment (ThomasGillespie) [reply code/segmentedtickerplant/datastripe.q]: I believe I added an extra `"` while neglecting a square bracket, thanks for the spot.

File: code/segmentedtickerplant/datastripe.q [hunk 3]
```diff
@@ -53,7 +53,7 @@ subsegment:{[tbl;segid];
       :()];
       
      //if a filter has been provided use subtablefiltered, if not use subtable for all syms
-     $[count filter;.ps.subtablefiltered[string[tbl];filter;""];.ps.subtable[string[tbl];"
+     $[count filter;.ps.subtablefiltered[string[tbl];filter;""];.ps.subtable[string[tbl]]
```
Comment (dmitchard) [inline code/segmentedtickerplant/datastripe.q:56]: Still missing 1 more ]

**Review by jonathonmcmurray — APPROVED** [review]

**Pull request merged**